model/labels/labels.go

```diff
@@ -248,6 +248,16 @@ func (ls Labels) WithoutEmpty() Labels {
 	return ls
 }
 
+// ByteSize returns the approximate size of the labels in bytes.
+// String and slice header size is ignored because it should be amortized to zero.
```

Comment:
I can accept the slice header might be re-used, but the string headers don't exist anywhere except this `Labels`, so should absolutely be counted.

Reply:
Update on taking string header into the calculation 05b44ab22d2832e38b01f305ecb3402fc270b702